hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -13,6 +13,160 @@
                      "hw/ip/tlul/data/tlul_testplan.hjson"]
 
   testpoints: [
+
+    //////////////////
+    // Host Device  //
+    // RV_CORE_IBEX //
+    //////////////////
+    //
+    // Note: the Debug Module will be referred to as RVDM.
+    {
+      name: chip_ibex_debug_basic
```

Comment:
should be covered by [rom_e2e_debug](https://github.com/lowRISC/opentitan/blob/a00ce54bf6341a296c0d3864008487e0b65e5787/sw/device/silicon_creator/rom/data/rom_testplan.hjson#L787) and [rom_e2e_jtag_inject](https://github.com/lowRISC/opentitan/blob/a00ce54bf6341a296c0d3864008487e0b65e5787/sw/device/silicon_creator/rom/data/rom_testplan.hjson#L816)

Reply:
No action needed - resolving.